model/src/main/kotlin/config/AdvisorConfiguration.kt

```diff
@@ -50,7 +50,7 @@ data class GitHubDefectsConfiguration(
      * The access token to authenticate against the GitHub GraphQL endpoint.
      */
     @JsonProperty(access = JsonProperty.Access.WRITE_ONLY)
-    val token: String?,
+    val token: String? = null,
```

Comment:
I believe this property is not optional, and should not have been nullable to start with. Can you confirm @oheger-bosch?

Reply:
Can this be considered out of scope of this commit / PR?